taxi/forms.py

```diff
@@ -0,0 +1,47 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm, UserChangeForm
+from django.core.exceptions import ValidationError
+from django.forms import ModelForm
+
+from taxi.models import Driver, Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if len(license_number) != 8:

```

Comment:
1. Use MaxLengthValidator and RegexValidator here
2. Instead of using clean define a field like so:
```
license_number = forms.CharField(
        validators=[